src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -249,8 +248,12 @@ static void MarkQuantizedNodes(const Graph& src,
 Graph QuantizeGraph(Graph &&src) {
   static const auto& flist_outputs = nnvm::Op::GetAttr<nnvm::FListOutputNames>("FListOutputNames");
   static const auto& need_requantize_map = Op::GetAttr<mxnet::FNeedRequantize>("FNeedRequantize");
```

Comment:
Maybe clang format whole file? Or at least the lines around the code you changed/added?

Reply:
I didn't want to format whole files to keep change clean and easy reviewable - I have applied formatter to my changes